src/Tools/Source/RunTests/TestRunner.cs

```diff
@@ -70,6 +70,7 @@ internal async Task<RunAllResult> RunAllAsync(IEnumerable<AssemblyInfo> assembly
                             if (!testResult.Succeeded)
                             {
                                 failures++;
+                                ConsoleUtil.WriteLine(ConsoleColor.Red, "Test failure log: " + testResult.ResultsFilePath);
```

Comment:
Considering adding the red output coloring to the `catch` block just below as well.